src/pages/Deeplink.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <div
```

Comment:
Include breadcrumbs component:

```html
<template>
   <div id="deeplink">
      <breadcrumbs :name="$t('sidebar.deeplink')" />
   </div>
</template>

<script>
import Breadcrumbs from "@/components/Breadcrumbs";

export default {
   //...
   components: {
      Breadcrumbs,
   },
  //...
}
</script>
```

Also consider reusing the dashboard ui elements for theme consitency